.storybook/preview.tsx

```diff
@@ -0,0 +1,35 @@
+import type { Preview } from "@storybook/react-vite";
+import * as React from "react";
+
+import "./main.css";
+import { themes } from "storybook/theming";
+
+const preview: Preview = {
+  tags: ["autodocs", "!dev"],
+  decorators: [
+    (Story) => (
+      <div className="bg-white dark:bg-gray-950 p-8 flex justify-center">
+        <Story />
+      </div>
+    ),
+  ],
+  parameters: {
+    layout: "fullscreen",
+    darkMode: {
+      darkClass: "dark",
+      lightClass: "light",
+      stylePreview: true,
+      dark: themes.light,
+      light: themes.light,
```

Comment:
I only wanted to toggle dark-mode for actual components and not the whole interface.